lms/static/scripts/frontend_apps/components/test/ValidationMessage-test.js

```diff
@@ -19,13 +19,13 @@ describe('ValidationMessage', () => {
 
   it('renders closed by default', () => {
     const wrapper = renderMessage();
-    assert.isTrue(wrapper.find('.ValidationMessage--closed').exists());
+    assert.isTrue(wrapper.find('[data-testid="message-closed"]').exists());
```

Comment:
Maybe you could create a variable for this selector?

Reply:
Yes, that would be better. Missed that one.

Reply:
Oh, actually, it's only used twice, so I don't know that it's worth it.